src/main/java/seedu/address/logic/commands/MarkTutorialCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tutorial.Tutorial;
+
+
+/**
+ * Marks a tutorial as complete.
+ */
+public class MarkTutorialCommand extends Command {
+
+    public static final String COMMAND_WORD = "marktut";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks a tutorial as complete "
+            + "by the index number used in the last person listing.\n "
+            + "Parameters: INDEX (must be a positive integer)\n "
+            + "Example: "
+            + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_MARKTUT_SUCCESS = "Marked Tutorial %1$s";
+
+    private final Index index;
+
+    /**
+     * Creates a MarkTutorialCommand to mark the tutorial
+     *
+     * @param index The position of the tutorial in the tutorial list
+     */
+    public MarkTutorialCommand(Index index) {
+        requireAllNonNull(index);
+
+        this.index = index;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+
+        if (other == this) {
+            return true;
+        }
+
+        if (!(other instanceof MarkTutorialCommand)) {
+            return false;
+        }
+
+        MarkTutorialCommand e = (MarkTutorialCommand) other;
+        return index.equals(e.index);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        List<Tutorial> lastShownList = model.getFilteredTutorialList();
```

Comment:
Maybe there should be a `requireNonNull(model);` line here

Reply:
Added for additional safety, thank you for pointing out!